httpserver/rust/src/httpserver.rs

```diff
@@ -372,6 +383,18 @@ pub fn decode_http_response(
     };
     Ok(__result)
 }
+#[async_trait]
+pub trait WebsocketClient {
+    /// returns the capability contract id for this interface
+    fn contract_id() -> &'static str {
+        "wasmcloud:httpserver:ws"
+    }
```

Comment:
Since this will be an entirely new contract, we _might_ need to create either a separate capability provider in the current wasmCloud model _or_ redefine how we're thinking of websockets 🤔